examples/research_projects/wav2vec2/run_common_voice.py

```diff
@@ -407,6 +417,38 @@ def speech_file_to_array_fn(batch):
         remove_columns=eval_dataset.column_names,
         num_proc=data_args.preprocessing_num_workers,
     )
+    
+    
+    # augmenting the datasets.
+    # We need to read the aduio files as arrays, minupulate the audio randomly and tokenize the targets.
+    augment = Compose([
```

Comment:
This should be done optionally IMO and not be the default -> could we add an argument `args.augment_speech_data`?

Reply:
Yes
What about the mass of augmented data ?
Maybe another argument how much percentage should be resampled ?

Reply:
I'm fine with that!